flipperdevices/flipperzero-firmware
# Decode RAW recordings

# What's new

![Flipper_SubGhz_RAW_decoder-2022-08-26_21 36 43](https://user-images.githubusercontent.com/678500/186979381-e55fc41e-c1ac-45ab-b0cf-6c1d08baf74a.gif)

- Add `decode` to the More menu of saved RAW recordings
- Reuses the SubGhz Receiver View
- Tweaked this view to support:
  - Live versus File view
  - File progress (instead of Config button) in left bottom
- Reuses the SubGhz Receiver Info scene
- Add `subghz_file_encoder_worker_get_text_progress`

The icon could use a little improvement. Unfortunately I'm not a fluent designer.
assets/icons/SubGhz/Decoding_123x52.png

# Verification 

- Make a RAW recording of signals supported by available decoders
- Save the RAW recording
- From a saved RAW recording, open the `More` menu
- Select the `Decode` option
- See how Flipper decodes the recorded signals

# Checklist (For Reviewer)
